pyproject.toml

```diff
@@ -10,6 +10,7 @@ flask = "^1.1.2"
 python-dotenv = "^0.14.0"
 requests = "^2.27.1"
 pytest = "^7.1.1"
```

Comment:
If you moved dependencies like pytest (things you don't need in production) down into the dev dependencies section, then you could skip installing them in the production version of your image.